rust/connlib/clients/shared/src/control.rs

```diff
@@ -278,13 +278,7 @@ impl<CB: Callbacks + 'static> ControlPlane<CB> {
         match (reply_error.error, reference) {
             (ErrorInfo::Offline, Some(reference)) => {
                 let Ok(resource_id) = reference.parse::<ResourceId>() else {
-                    tracing::error!(
-                        "An offline error came back with a reference to a non-valid resource id"
-                    );
-                    let _ = self
-                        .tunnel
-                        .callbacks()
-                        .on_error(&Error::ControlProtocolError);
+                    tracing::warn!("The portal responded with an Offline error. Is the Resource associated with any online Gateways? ResourceId: {reference}");
                     return Ok(());
```

Comment:
the `reference` no longer points to a resource id

Reply:
👍